src/styles.jl

```diff
@@ -48,6 +48,27 @@ function styles_cell_xf_numFmtId(wb::Workbook, index::Int) :: Int
     return parse(Int, el["numFmtId"])
 end
 
+function styles_add_numFmt(wb::Workbook, format_code::AbstractString) :: Integer
+    xroot = styles_xmlroot(wb)
+
+    numfmts = find(xroot, "/xpath:styleSheet/xpath:numFmts", SPREADSHEET_NAMESPACE_XPATH_ARG)
+    if isempty(numfmts)
+        stylesheet = findfirst(xroot, "/xpath:styleSheet", SPREADSHEET_NAMESPACE_XPATH_ARG)
+        numfmts = EzXML.addelement!(stylesheet, "numFmts")
+    else
+        numfmts = numfmts[1]
+    end
+
+    existing_numFmt_elements_count = EzXML.countelements(numfmts)
+
+    new_fmt = EzXML.addelement!(numfmts, "numFmt")
+    fmt_code = 164 + existing_numFmt_elements_count
+    new_fmt["numFmtId"] = fmt_code
+    new_fmt["formatCode"] = format_code
+
+    return fmt_code
+end
```

Comment:
Maybe a Julia type should be made to represent this?

Reply:
I will be posting a refactoring PR in the near future, so I'm holding off on adding a new type until then.